### PR TITLE
Mv3: Remove UIkit and revamp pages with FastForward CSS

### PR DESCRIPTION
**Pull request by NotAProton**
To test before navigate pages, click on these links with the extension installed:
crowd-bypassed: https://fastforward.team/bypassed?target=https%3A%2F%2Fgoogle.com&crowd=true
before-navigate: https://fastforward.team/bypassed?target=https%3A%2F%2Fgoogle.com

Description:
- Removed UIkit, now the extension uses no external libraries 
- Rewrote options, before-navigate and crowd-bypassed pages using css from the FastForward website
- Copied over CONTRIBUTORS.md from main

Checklist:

- [x] I made sure there are no unnecessary changes in the code*
- [x] Tested on Chromium- Chrome 112 Windows
- [x] Tested on Firefox

### CODE REVIEW

**Comment by AliahX** [comment]
i'll check it out in a minute, just working on another pr rn

File: src/html/base.js [hunk 1]
```diff
@@ -1,58 +1,48 @@
...
+          cancel();
+          callback();
+          return;
+        }
+        titleElm.textContent = '(' + secondsLeft + ') ' + title;
+        if (secondsLeft == 1) {
+          p.textContent = brws.i18n.getMessage(message + 'Singular') + ' ';
+        } else {
+          p.textContent =
+            brws.i18n.getMessage(message).replace('%', secondsLeft) + ' ';
```
[the remark this answers is not in the document]
Comment (NotAProton) [reply src/html/base.js]: That is actually redundant and should have been removed in #921

[the remark this answers is not in the document]
Comment (AliahX) [reply src/html/base.js]: oh oki

**Comment by AliahX** [comment]
This branch is missing the CONTRIBUTORS.md file that is linked to by the extension.

**Comment by NotAProton** [comment]
<details>
  <summary>Resolved</summary>

what about mobile responsiveness?
chrome-extension___ambaeodiomeedcpomkgkkobenfgddedg_html_options html
</details>

**Comment by AliahX** [comment]
I think i have a solution maybe. Do we need the changelog faq and contributors buttons on mobile?

**Comment by NotAProton** [comment]
I think only 3 people ever clicked on them lol, we can just push them to the bottom

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Thanks to everyone who helps make FastForward:
+
+- The [code contributors](https://github.com/FastForwardTeam/FastForward/graphs/contributors) right here at Github
+- The [website devs](https://github.com/FastForwardTeam/Website/graphs/contributors)
+- The translators over at [Crowdin](https://crowdin.com/project/fastforward)
+
+## Libraries used by FastForward
+
+- FastForward uses [UIkit](https://getuikit.com/) for its design
```
Comment (ghost) [inline CONTRIBUTORS.md]: Remove these 2 lines, that's the whole point of the PR.

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**